include/boost/multiprecision/mpfr.hpp

```diff
@@ -317,8 +323,8 @@ struct mpfr_float_imp<digits10, allocate_dynamic>
          return *this;
       }
 
-      BOOST_MP_ASSERT(!(boost::math::isinf)(a));
-      BOOST_MP_ASSERT(!(boost::math::isnan)(a));
+      BOOST_MP_ASSERT(!(boost::multiprecision::detail::isinf)(a));
```

Comment:
Another where an update to BOOST_MP_FLOAT128_USING and unqualified call fixes things.